### ISSUE TITLE
CPU-only Milvus with Intel MKL built from source crashed when running python example

### ISSUE THREAD

**Issue by caosiyang** [milvus-io/milvus#2202]
**Describe the bug**
CPU-only Milvus **with Intel MKL** built from source crashed when running [python example](https://raw.githubusercontent.com/milvus-io/pymilvus/0.8.0/examples/example.py).

The default OpenBLAS version has no problem.

**Steps/Code to reproduce behavior**
- install Intel MKL `yum install intel-mkl-2019.5-075`
- build Milvus `./build.sh -m -t Release`

**Expected behavior**
Pass the [python example](https://raw.githubusercontent.com/milvus-io/pymilvus/0.8.0/examples/example.py).

**Environment details**
- CentOS Linux release 7.6.1810 (Core)
- Method of installation (built from source with Intel MKL 2019.5-075)
- Milvus version (CPU-only v0.8.0 [44e8e0206537046ccd89306951ca704a545e0417](https://github.com/milvus-io/milvus/tree/44e8e0206537046ccd89306951ca704a545e0417)) 
- Milvus configuration (Settings you made in `server_config.yaml`)
	```
	version: 0.3
	
	server_config:
	  address: 0.0.0.0
	  port: 19530
	  deploy_mode: single
	  time_zone: UTC+8
	  web_port: 19121
	
	db_config:
	  backend_url: sqlite://:@:/
	  preload_collection:
	  auto_flush_interval: 1
	
	storage_config:
	  primary_path: /tmp/milvus
	  secondary_path:
	
	metric_config:
	  enable_monitor: false
	  address: 127.0.0.1
	  port: 9091
	
	cache_config:
	  cpu_cache_capacity: 4
	  insert_buffer_size: 1
	  cache_insert_data: false
	
	engine_config:
	  use_blas_threshold: 1100
	  gpu_search_threshold: 1000
	
	gpu_resource_config:
	  enable: false
	  cache_capacity: 1
	  search_resources:
	    - gpu0
	  build_index_resources:
	    - gpu0
	
	tracing_config:
	  json_config_path:
	
	wal_config:
	  enable: true
	  recovery_error_ignore: true
	  buffer_size: 256
	  wal_path: /tmp/milvus/wal
	
	```

**Screenshots**

- start_server.sh output

	```
	
	    __  _________ _   ____  ______    
	   /  |/  /  _/ /| | / / / / / __/    
	  / /|_/ // // /_| |/ / /_/ /\ \    
	 /_/  /_/___/____/___/\____/___/     
	
	Welcome to use Milvus!
	Milvus Release version: v0.8.0, built at 2020-04-30 16:17.30, with MKL library.
	You are using Milvus CPU edition
	Last commit id: 44e8e0206537046ccd89306951ca704a545e0417
	
	Loading configuration from: /root/test/milvus/conf/server_config.yaml
	Initializing log config from: /root/test/milvus/conf/log_config.conf
	Milvus server started successfully!
	Milvus server is going to shutdown ...
	Milvus server is going to shutdown ...
	terminate called after throwing an instance of 'std::system_error'
	  what():  Invalid argument
	./start_server.sh: line 7: 28570 Aborted                 (core dumped) $milvus_path/bin/milvus_server -c $milvus_path/conf/server_config.yaml -l $milvus_path/conf/log_config.conf
	```

- global.log

	```
	[2020-04-30 16:22:33,485][INFO][SERVER][Start][milvus_server] Milvus Release version: v0.8.0, built at 2020-04-30 16:17.30
	[2020-04-30 16:22:33,485][INFO][SERVER][Start][milvus_server] CPU edition
	[2020-04-30 16:22:33,487][INFO][SERVER][LogConfigInFile][milvus_server] 
	
	***************Config in file***************
	
	version: 0.3
	server_config:
	  address: 0.0.0.0
	  port: 19530
	  deploy_mode: single
	  time_zone: UTC+8
	  web_port: 19121
	db_config:
	  backend_url: sqlite://:@:/
	  preload_collection: ~
	  auto_flush_interval: 1
	storage_config:
	  primary_path: /tmp/milvus
	  secondary_path: ~
	metric_config:
	  enable_monitor: false
	  address: 127.0.0.1
	  port: 9091
	cache_config:
	  cpu_cache_capacity: 4
	  insert_buffer_size: 1
	  cache_insert_data: false
	engine_config:
	  use_blas_threshold: 1100
	  gpu_search_threshold: 1000
	gpu_resource_config:
	  enable: false
	  cache_capacity: 1
	  search_resources:
	    - gpu0
	  build_index_resources:
	    - gpu0
	tracing_config:
	  json_config_path: ~
	wal_config:
	  enable: true
	  recovery_error_ignore: true
	  buffer_size: 256
	  wal_path: /tmp/milvus/wal
	[2020-04-30 16:22:33,487][INFO][SERVER][LogCpuInfo][milvus_server] 
	
	***************CPU***************
	
	processor	: 3
	vendor_id	: GenuineIntel
	cpu family	: 6
	model		: 94
	model name	: Intel Core Processor (Skylake)
	stepping	: 3
	microcode	: 0x1
	cpu MHz		: 2194.916
	cache size	: 16384 KB
	physical id	: 1
	siblings	: 2
	core id		: 0
	cpu cores	: 1
	apicid		: 3
	initial apicid	: 3
	fpu		: yes
	fpu_exception	: yes
	cpuid level	: 13
	wp		: yes
	flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology eagerfpu pni pclmulqdq ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm rdseed adx smap xsaveopt arat
	bogomips	: 4389.83
	clflush size	: 64
	cache_alignment	: 64
	address sizes	: 46 bits physical, 48 bits virtual
	power management:
	
	
	[2020-04-30 16:22:33,487][INFO][SERVER][LogConfigInMem][milvus_server] 
	
	***************Config in memory***************
	
	{
	   "cache_config": {
	      "cache_insert_data": "false",
	      "cpu_cache_capacity": "4",
	      "cpu_cache_threshold": "0.7",
	      "insert_buffer_size": "1"
	   },
	   "db_config": {
	      "archive_days_threshold": "0",
	      "archive_disk_threshold": "0",
	      "auto_flush_interval": "1",
	      "backend_url": "sqlite://:@:/",
	      "preload_collection": ""
	   },
	   "engine_config": {
	      "omp_thread_num": "0",
	      "use_avx512": "true",
	      "use_blas_threshold": "1100"
	   },
	   "metric_config": {
	      "address": "127.0.0.1",
	      "enable_monitor": "false",
	      "port": "9091"
	   },
	   "server_config": {
	      "address": "0.0.0.0",
	      "deploy_mode": "single",
	      "port": "19530",
	      "time_zone": "UTC+8",
	      "web_port": "19121"
	   },
	   "storage_config": {
	      "primary_path": "/tmp/milvus",
	      "s3_access_key": "minioadmin",
	      "s3_address": "127.0.0.1",
	      "s3_bucket": "milvus-bucket",
	      "s3_enable": "false",
	      "s3_port": "9000",
	      "s3_secret_key": "minioadmin",
	      "secondary_path": ""
	   },
	   "tracing_config": {
	      "json_config_path": ""
	   },
	   "wal_config": {
	      "buffer_size": "256",
	      "enable": "true",
	      "recovery_error_ignore": "true",
	      "wal_path": "/tmp/milvus/wal"
	   }
	}
	[2020-04-30 16:22:33,491][INFO][ENGINE][Build][milvus_server] Using SQLite
	[2020-04-30 16:22:33,756][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 47014792399064
	[2020-04-30 16:22:34,760][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 54951000
	[2020-04-30 16:22:35,760][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 54951000
	[2020-04-30 16:22:36,760][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 54951000
	[2020-04-30 16:22:37,761][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 54951000
	[2020-04-30 16:22:38,110][INFO][SERVER][PutToQueue][grpcpp_sync_ser] Create new thread for request group: info
	[2020-04-30 16:22:38,120][INFO][SERVER][PutToQueue][grpcpp_sync_ser] Create new thread for request group: ddl_dml
	[2020-04-30 16:22:38,120][INFO][WAL][CreateCollection][reqsched_thread] create collection example_collection_ 0
	[2020-04-30 16:22:38,761][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 54951000
	[2020-04-30 16:22:38,883][INFO][SERVER][Insert][grpcpp_sync_ser] [insert][0] Start insert.
	[2020-04-30 16:22:38,888][INFO][SERVER][OnExecute][reqsched_thread] [insert][0] Execute insert request.
	[2020-04-30 16:22:38,899][INFO][WAL][Insert][reqsched_thread] [insert][0]example_collection_ insert in part  with lsn 5200040
	[2020-04-30 16:22:38,899][INFO][WAL][GetNextRecord][wal_thread] record type 2 collection example_collection_ lsn 5200040
	[2020-04-30 16:22:38,902][INFO][SERVER][Insert][grpcpp_sync_ser] [insert][0] Insert done.
	[2020-04-30 16:22:38,920][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 5200040
	[2020-04-30 16:22:38,954][INFO][WAL][Flush][reqsched_thread] example_collection_ want to be flush, lsn 5200040
	[2020-04-30 16:22:38,954][INFO][WAL][operator()][wal_thread] record flush collection example_collection_ lsn 5200040
	[2020-04-30 16:22:38,975][INFO][WAL][CollectionFlushed][wal_thread] example_collection_ is flushed by lsn 5200040
	[2020-04-30 16:22:38,975][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 5200040
	[2020-04-30 16:22:38,985][INFO][WAL][Flush][reqsched_thread]  want to be flush, lsn 0
	[2020-04-30 16:22:39,761][INFO][WAL][GetNextRecord][wal_thread] record type 0 collection  lsn 54951000
	[2020-04-30 16:22:39,779][INFO][SERVER][HandleSignal][taskexector_th] Server received critical signal: 11
	[2020-04-30 16:22:39,779][INFO][SERVER][PrintStacktrace][taskexector_th] Call stack:
	[2020-04-30 16:22:39,779][INFO][SERVER][HandleSignal][taskexector_th] Server received critical signal: 11
	[2020-04-30 16:22:39,779][INFO][SERVER][PrintStacktrace][taskexector_th] Call stack:
	[2020-04-30 16:22:39,779][INFO][SERVER][PrintStacktrace][taskexector_th] /root/debug/milvus/bin/milvus_server() [0x63572c]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /root/debug/milvus/bin/milvus_server() [0x636080]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /lib64/libc.so.6(+0x36340) [0x2ac26ae2c340]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /root/debug/milvus/bin/milvus_server() [0x1cf2ae0]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /root/debug/milvus/bin/milvus_server() [0x63572c]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /root/debug/milvus/bin/milvus_server() [0x636080]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /lib64/libc.so.6(+0x36340) [0x2ac26ae2c340]
	[2020-04-30 16:22:39,780][INFO][SERVER][PrintStacktrace][taskexector_th] /root/debug/milvus/bin/milvus_server() [0x1cf2ad5]
	```

- debug.log

	```
	[2020-04-30 16:22:33,488][DEBUG][ENGINE][Initialize][milvus_server] FAISS hook AVX2
	[2020-04-30 16:22:33,756][DEBUG][WAL][Init][milvus_server] start_lsn 0 end_lsn 0
	[2020-04-30 16:22:34,759][DEBUG][ENGINE][SetGlobalLastLSN][wal_thread] Update global lsn = 0
	[2020-04-30 16:22:38,110][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 0
	[2020-04-30 16:22:38,110][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] HasCollectionRequest(collection=example_collection_): totally cost (0.000398 second [0.397730 ms])
	[2020-04-30 16:22:38,119][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 1
	[2020-04-30 16:22:38,120][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] CreateCollectionRequest(collection=example_collection_, dimension=128): check validation (0.000010 second [0.010301 ms])
	[2020-04-30 16:22:38,122][DEBUG][ENGINE][CreateCollection][reqsched_thread] Successfully create collection: example_collection_
	[2020-04-30 16:22:38,122][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] CreateCollectionRequest(collection=example_collection_, dimension=128): totally cost (0.002325 second [2.324527 ms])
	[2020-04-30 16:22:38,131][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 2
	[2020-04-30 16:22:38,132][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] ShowCollectionsRequest: totally cost (0.000294 second [0.293738 ms])
	[2020-04-30 16:22:38,136][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 3
	[2020-04-30 16:22:38,136][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] ShowCollectionInfoRequest(collection=example_collection_): totally cost (0.000592 second [0.592088 ms])
	[2020-04-30 16:22:38,142][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 4
	[2020-04-30 16:22:38,142][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] DescribeCollectionRequest(collection=example_collection_): totally cost (0.000174 second [0.174479 ms])
	[2020-04-30 16:22:38,883][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 5
	[2020-04-30 16:22:38,889][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] [insert][0] InsertRequest(collection=example_collection_, n=10000, partition_tag=): check validation (0.000257 second [0.257429 ms])
	[2020-04-30 16:22:38,889][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] [insert][0] InsertRequest(collection=example_collection_, n=10000, partition_tag=): prepare vectors data (0.000140 second [0.139874 ms])
	[2020-04-30 16:22:38,902][DEBUG][ENGINE][UpdateCollectionFlag][reqsched_thread] Successfully update collection flag, collection id = example_collection_
	[2020-04-30 16:22:38,902][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] [insert][0] InsertRequest(collection=example_collection_, n=10000, partition_tag=): add vectors to engine (0.013361 second [13.360568 ms])
	[2020-04-30 16:22:38,902][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] [insert][0] InsertRequest(collection=example_collection_, n=10000, partition_tag=): total cost (0.013788 second [13.787711 ms])
	[2020-04-30 16:22:38,911][DEBUG][ENGINE][CreateCollectionFile][wal_thread] Successfully create collection file, file id = 1588234958909705000
	[2020-04-30 16:22:38,916][DEBUG][ENGINE][Add][wal_thread] [insert][0]Insert into segment
	[2020-04-30 16:22:38,954][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 6
	[2020-04-30 16:22:38,954][DEBUG][SERVER][OnExecute][reqsched_thread] FlushRequest flush collections: example_collection_, 
	[2020-04-30 16:22:38,954][DEBUG][ENGINE][Flush][reqsched_thread] Begin flush collection: example_collection_
	[2020-04-30 16:22:38,954][DEBUG][ENGINE][Flush][reqsched_thread] WAL flush
	[2020-04-30 16:22:38,955][DEBUG][ENGINE][Flush][wal_thread] Flushing collection: example_collection_
	[2020-04-30 16:22:38,956][DEBUG][SERVER][PrintTimeRecord][wal_thread] SegmentWriter::WriteBloomFilter: Initializing bloom filter (0.001379 second [1.378868 ms])
	[2020-04-30 16:22:38,964][DEBUG][SERVER][PrintTimeRecord][wal_thread] SegmentWriter::WriteBloomFilter: Adding 10000 ids to bloom filter (0.007763 second [7.762560 ms])
	[2020-04-30 16:22:38,964][DEBUG][SERVER][PrintTimeRecord][wal_thread] SegmentWriter::WriteBloomFilter: Writing bloom filter (0.000166 second [0.165778 ms])
	[2020-04-30 16:22:38,964][DEBUG][SERVER][PrintTimeRecord][wal_thread] SegmentWriter::Serialize: Writing bloom filter done (0.009377 second [9.377033 ms])
	[2020-04-30 16:22:38,969][DEBUG][SERVER][PrintTimeRecord][wal_thread] write vectors: write rv done (0.005211 second [5.210730 ms])
	[2020-04-30 16:22:38,970][DEBUG][SERVER][PrintTimeRecord][wal_thread] write vectors: write uids done (0.000292 second [0.291565 ms])
	[2020-04-30 16:22:38,970][DEBUG][SERVER][PrintTimeRecord][wal_thread] SegmentWriter::Serialize: Writing vectors and uids done (0.005690 second [5.689594 ms])
	[2020-04-30 16:22:38,970][DEBUG][SERVER][PrintTimeRecord][wal_thread] SegmentWriter::Serialize: Writing deleted docs done (0.000105 second [0.104643 ms])
	[2020-04-30 16:22:38,972][DEBUG][ENGINE][UpdateCollectionFile][wal_thread] Update single collection file, file id = 1588234958909705000
	[2020-04-30 16:22:38,972][DEBUG][ENGINE][Serialize][wal_thread] New raw file 1588234958909705000 of size 5120000 bytes, lsn = 5200040
	[2020-04-30 16:22:38,972][DEBUG][ENGINE][Serialize][wal_thread] Flushed segment 1588234958909705000
	[2020-04-30 16:22:38,974][DEBUG][ENGINE][UpdateCollectionFlushLSN][wal_thread] Successfully update collection flush_lsn, collection id = example_collection_ flush_lsn = 5200040
	[2020-04-30 16:22:38,975][DEBUG][SERVER][PrintTimeRecord][wal_thread] MemTable::Serialize collection example_collection_: Finished flushing (0.019796 second [19.795844 ms])
	[2020-04-30 16:22:38,975][DEBUG][ENGINE][Flush][wal_thread] Flushed collection: example_collection_
	[2020-04-30 16:22:38,975][DEBUG][ENGINE][Flush][reqsched_thread] End flush collection: example_collection_
	[2020-04-30 16:22:38,975][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] FlushRequest flush collections: example_collection_, : totally cost (0.020949 second [20.949069 ms])
	[2020-04-30 16:22:38,978][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 7
	[2020-04-30 16:22:38,979][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] CountCollectionRequest(collection=example_collection_): check validation (0.000253 second [0.253344 ms])
	[2020-04-30 16:22:38,979][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] CountCollectionRequest(collection=example_collection_): totally cost (0.000772 second [0.771919 ms])
	[2020-04-30 16:22:38,984][DEBUG][SERVER][OnPostRecvInitialMetaData][grpcpp_sync_ser] milvus generate request_id: 8
	[2020-04-30 16:22:38,985][DEBUG][SERVER][PrintTimeRecord][reqsched_thread] CreateIndexRequest(collection=example_collection_): check validation (0.000334 second [0.334289 ms])
	[2020-04-30 16:22:38,985][DEBUG][ENGINE][Flush][reqsched_thread] Begin flush all collections
	[2020-04-30 16:22:38,985][DEBUG][ENGINE][Flush][reqsched_thread] WAL flush
	[2020-04-30 16:22:38,985][DEBUG][ENGINE][Flush][reqsched_thread] End flush all collections
	[2020-04-30 16:22:38,985][DEBUG][ENGINE][DropIndex][reqsched_thread] Drop index for collection: example_collection_
	[2020-04-30 16:22:38,985][DEBUG][ENGINE][DropCollectionIndexRecursively][reqsched_thread] Drop index for collection: example_collection_
	[2020-04-30 16:22:38,985][DEBUG][ENGINE][DropCollectionIndex][reqsched_thread] Successfully drop collection index, collection id = example_collection_
	[2020-04-30 16:22:38,987][DEBUG][ENGINE][UpdateCollectionIndex][reqsched_thread] Successfully update collection index, collection id = example_collection_
	[2020-04-30 16:22:38,988][DEBUG][ENGINE][FilesByType][reqsched_thread] Get collection files by type. raw files:1 new files:0 new_merge files:0 new_index files:0 to_index files:0
	[2020-04-30 16:22:38,988][DEBUG][ENGINE][WaitCollectionIndexRecursively][reqsched_thread] Non index files detected! Will build index 1
	[2020-04-30 16:22:38,989][DEBUG][ENGINE][UpdateCollectionFilesToIndex][reqsched_thread] Update files to to_index, collection id = example_collection_
	[2020-04-30 16:22:39,757][DEBUG][ENGINE][FilesToIndex][milvus_server] Collect 1 to-index files
	[2020-04-30 16:22:39,758][DEBUG][ENGINE][BackgroundBuildIndex][milvus_server] Background build index thread begin
	[2020-04-30 16:22:39,758][DEBUG][ENGINE][MarkOngoingFileNoLock][milvus_server] Mark ongoing file:1588234958909705000 refcount:1
	[2020-04-30 16:22:39,758][DEBUG][SERVER][AddToIndexFiles][milvus_server] BuildIndexJob 0 add to_index file: 1, location: /tmp/milvus/db/tables/example_collection_/1588234958909705000/1588234958909705000
	[2020-04-30 16:22:39,758][DEBUG][ENGINE][Build][jobmgr_thread] EngineFactory index type: 1
	[2020-04-30 16:22:39,758][DEBUG][ENGINE][ExecutionEngineImpl][jobmgr_thread] Index params: {"dim":128,"gpu_id":0,"metric_type":"L2","nlist":2048}
	[2020-04-30 16:22:39,758][DEBUG][SERVER][Run][jobmgr_thread] FallbackPass!
	[2020-04-30 16:22:39,758][DEBUG][SERVER][loader_function][taskloader_th] cpu load BuildIndexTask
	[2020-04-30 16:22:39,758][DEBUG][ENGINE][Load][taskloader_th] Index params: {"dim":128,"gpu_id":0,"metric_type":"L2"}
	[2020-04-30 16:22:39,764][DEBUG][ENGINE][Load][taskloader_th] set uids 10000 for index /tmp/milvus/db/tables/example_collection_/1588234958909705000/1588234958909705000
	[2020-04-30 16:22:39,769][DEBUG][ENGINE][Load][taskloader_th] Finished loading raw data from segment /tmp/milvus/db/tables/example_collection_/1588234958909705000
	[2020-04-30 16:22:39,769][DEBUG][SERVER][PrintTimeRecord][taskloader_th] XBuildIndexTask::Load: Build index task load file id:1 DISK2CPU file type:2 size:5201250 bytes from location: /tmp/milvus/db/tables/example_collection_/1588234958909705000/1588234958909705000 totally cost (0.010264 second [10.264269 ms])
	[2020-04-30 16:22:39,771][DEBUG][ENGINE][CreateCollectionFile][taskexector_th] Successfully create collection file, file id = 1588234959769572000
	[2020-04-30 16:22:39,771][DEBUG][ENGINE][Execute][taskexector_th] Begin build index for file:/tmp/milvus/db/tables/example_collection_/1588234958909705000/1588234959769572000
	[2020-04-30 16:22:39,771][DEBUG][ENGINE][BuildIndex][taskexector_th] Build index file: /tmp/milvus/db/tables/example_collection_/1588234958909705000/1588234959769572000 from: /tmp/milvus/db/tables/example_collection_/1588234958909705000/1588234958909705000
	[2020-04-30 16:22:39,771][DEBUG][ENGINE][BuildIndex][taskexector_th] Index params: {"dim":128,"gpu_id":0,"metric_type":"L2","nlist":2048,"rows":10000}
	[2020-04-30 16:22:39,771][DEBUG][ENGINE][BuildIndex][taskexector_th] Index config: {"dim":128,"gpu_id":0,"metric_type":"L2","nlist":163,"rows":10000}
	```

**Additional context**

Before starting Milvus, the data directory `/tmp/milvus` is empty.
It seems that Milvus crashed when building index.

**Comment by tinkerlin** [comment]
we will fix it in future release. 
thx :)

**Comment by shengjun1985** [comment]
Inactive. Close.

**Comment by qiaoyuKs** [comment]
where is the patch that fixed the bug, same problem with 1.1.1
